runtime/environment-cpu.yml

```diff
@@ -31,10 +31,12 @@ dependencies:
   - tensorflow-cpu=2.10.0
   - tqdm=4.64.1
   - xgboost=1.7.1
+  - future=0.18.2
   - pip:
       - flwr[simulation]==1.1.0
       - gower==0.1.2
       - interpret==0.3.0
       - git+https://github.com/ibarrond/Pyfhel.git@0dee7a77469627c4de4f2dca97cdc68bf006a77d
       - tensorflow_privacy==0.7.3
       - tensorflow-probability==0.15.0
+      - torchmetrics==0.11.0
```

Comment:
It looks like torchmetrics v0.11.0 should also be available through conda-forge. We have a preference for specifying dependencies with conda whenever possible for more reliable dependency resolution.

Reply:
Okay, I've changed it to be a dependency under conda so we can avoid any dependency resolution issues